tests/unit/data_manager/conftest.py

```diff
@@ -0,0 +1,9 @@
+import mongomock
+import pymongo
+import pytest
+
+
+@pytest.fixture
+@mongomock.patch(servers=(('server.example.com', 27017),))
```

Comment:
We are gonna have to decide whether we use `Fakes` or `mock`s.
Generally, we do not need both unless I am ignorant or missing something.